CriticalMass/SettingsSwitchTableViewCell.swift

```diff
@@ -7,18 +7,16 @@
 
 import UIKit
 
-protocol SettingsSwitchCellConfigurable {
-    func configure(isOn: Bool, handler: SettingsSwitchHandler?)
+protocol Switchable {
```

Comment:
Is `Switchable` to keep the SettingsController more adjustable and maybe add other switch cell later?

Reply:
Yes, my main though replace the repetitive code in SettingsViewController with a common interface

Reply:
Ok 👌